webui/react/src/pages/Settings/Settings.tsx

```diff
@@ -10,16 +10,20 @@ import { paths } from 'routes/utils';
 
 const { TabPane } = Tabs;
 
-enum TabType {
-  Account = 'account',
-  UserManagement = 'user-management',
+export enum TabType {
+  Account = 'Account',
+  UserManagement = 'User Management',
 }
 
 interface Params {
   tab?: TabType;
 }
 
-const DEFAULT_TAB_KEY = TabType.Account;
+export const TAB_KEYS = {
+  [TabType.Account]: 'account',
+  [TabType.UserManagement]: 'user-management',
+};
+export const DEFAULT_TAB_KEY = TabType.Account;
```

Comment:
minor issue, but do these two `const`s need to be exported?

Reply:
ah yeah, the 3rd test that was taken out used them. I'll remove the `export`. Thanks!